### ISSUE TITLE
Shall ID Number 7 should be applicable only to entropy sources for which the submitter is making an IID claim

### ISSUE THREAD

**Issue by joshuaehill** [usnistgov/90B-Shall-Statements#7]
This could be accomplished by changing the "Requirement Status" to

> Only Required if the submitter is making an IID claim.

or some equivalent change to the "Notes" entry.

**Comment by celic** [comment]
Agreed. The IID requirements are usually easy to tell if they are applicable, but pointing it out keeps things consistent.
